hpycc/connection.py

```diff
@@ -342,6 +357,11 @@ def run_ecl_string(self, string, syntax_check, delete_workunit):
             executed.
         delete_workunit: bool
             Delete workunit once completed.
+        stored : dict
+            Dictionary of key value pairs do to replace stored
```

Comment:
there is a typo here. Should also say dict or None. A few docstrings also need optional adding. Let's make sure the docstring is standardised